nightingale_dispatcher/src/nightingale_dispatcher/send_interface_request_task.py

```diff
@@ -2,12 +2,35 @@
 
 import rospy
 from nightingale_dispatcher.task import Task
+from nightingale_ros_bridge import BridgeConfig, UserInputs
+from nightingale_msgs.srv import InterfaceCall
 
 
 class SendInterfaceRequestTask(Task):
     def __init__(self):
         pass
 
     def execute(self, robot_state):
-        user_input = self.interface_comms_request()
-        return user_input
+        rospy.wait_for_service(BridgeConfig.UPDATE_UI_SERVICE, InterfaceCall)
```

Comment:
similar thing ive said before, but these wait for service can be a pain to debug. id add add rospy log statements around it (like i do in room_runner).

in the past, ive forgotten to run the service server, then my code just hangs and it comes down to these wait for service